github/resource_github_repository_collaborators_test.go

```diff
@@ -0,0 +1,538 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/google/go-github/v48/github"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccGithubRepositoryCollaborators(t *testing.T) {
+
+	var inOrgUser, inOrgUser2 string
+
+	if inOrgUser == "" || inOrgUser2 == "" {
+		t.Skip("set inOrgUser and inOrgUser2 to unskip this test run")
```

Comment:
@maroux one more thing, sorry! It looks like the way this test is written is that these values with never be set and thus the test will never be run. Is the intention to set these with environment variables?

Reply:
It's in line with https://github.com/integrations/terraform-provider-github/blob/main/github/resource_github_repository_collaborator_test.go#L13. I agree that an env var might be better suited here.

Reply:
Hey @maroux, thanks for the changes here! Would you be up for migrating this to an env var.  We'd rather not have this "pattern" propagated if we can help it.  If not, no worries; we need to unwind the other occurrence of this type of thing and will probably get to it then - but the preference would be to clean this up so that others don't run into the same discovery you have.  

Thanks again!  All the other changes look great; We're looking forward to merging them in.

Reply:
Done. I didn't put in any values in the workflow - please fill in as appropriate.